ts/components/wallet/card/CardComponent.style.ts

```diff
@@ -3,14 +3,26 @@ import { makeFontStyleObject } from "../../../theme/fonts";
 import variables from "../../../theme/variables";
 
 export default StyleSheet.create({
-  card: {
-    // iOS and Andorid card shadow
+  cardHeader:{
+    backgroundColor: variables.brandGray,
+    marginTop: -20,
+    borderRadius: 8,
+    marginLeft: 12,
+    marginRight: 12,
+    paddingBottom: 10,
+    borderBottomLeftRadius: 8,
```

Comment:
Why do you use both `borderBottomLeftRadius`/`borderBottomRightRadius` and `borderRadius`?